.github/actions/internal-triage-skip/action.yml

```diff
@@ -0,0 +1,30 @@
+---
+name: Skip Workflow if Labeled
+description: >
+    Skips the workflow if a label matches its filename
+    (e.g.  skip_aws_openshift_rosa_hcp_single_region_tests without the yml extension)
+
+outputs:
+    should_skip:
+        description: Indicates whether the workflow should be skipped
+        value: ${{ steps.check_labels.outputs.workflow_should_skip }}
+
+runs:
+    using: composite
+    steps:
+        - name: Get the current workflow filename and check for skip labels
+          id: check_labels
+          shell: bash
+          run: |
+              workflow_file_name=$(echo "${{ github.workflow_ref }}" | sed 's/@.*//')
+              workflow_file_name=$(basename "$workflow_file_name" | sed 's/\(.*\)\(\.yaml\|\.yml\)$/\1/')
+              echo "workflow_file_name=$workflow_file_name" | tee -a "$GITHUB_ENV"
```

Comment:
it's also part of the github context but can also be done your way.

`github.event.issue.labels.*.name`

Reply:
just alternative way to access the labels.